content/modules/redisearch/release-notes/redisearch-1.4-release-notes.md

```diff
@@ -389,7 +389,7 @@ This version improves overall stability and performance of RediSearch. It also d
 
 Like most search engines, RediSearch was designed for maintaining append-mostly indices. To update an existing document, the document is actually replaced - that is deleted and added to the index.
 
-Because RediSearch provides realtime indexing and searching it is very tempting to use for searching near-realtime data (or the products of its processing). In such cases, the number of indexed documents stays mostly static, but their contents are continuously updated.
+Because RediSearch provides realtime indexing and searching, it's sometimes used to search near-realtime data (or the results of its processing). In such cases, the number of indexed documents stays mostly static, but their contents are continuously updated.
```

Comment:
```suggestion
Because RediSearch provides realtime indexing and searching, it is sometimes used to search near-realtime data (or the results of its processing). In such cases, the number of indexed documents stays mostly static, but their contents are continuously updated.
```
I prefer to avoid contractions whenever possible.